Source/FieldSolver/SpectralSolver/SpectralSolver.H

```diff
@@ -89,10 +89,40 @@ class SpectralSolver
          * \brief Transform the component `i_comp` of MultiFab `mf`
          *  to spectral space, and store the corresponding result internally
          *  (in the spectral field specified by `field_index`) */
```

Comment:
This is slightly orthogonal to this PR, but could you add the missing documentation for the input parameters (including `stag`)?